core/src/main/java/org/apache/carbondata/core/constants/CarbonCommonConstants.java

```diff
@@ -573,23 +573,44 @@
   /**
    * NUMBER_OF_TRIES_FOR_LOAD_METADATA_LOCK
    */
-  public static final int NUMBER_OF_TRIES_FOR_LOAD_METADATA_LOCK_DEFAULT = 3;
+  public static final int NUMBER_OF_TRIES_FOR_CONCURRENT_LOCK_DEFAULT = 3;
```

Comment:
timeout check should be MAX_TIMEOUT_FOR_CONCURRENT_LOCK_DEFAULT = 1 sec, 
NUMBER_OF_TRIES_FOR_CONCURRENT_LOCK  = 100;
also write comment why bigger number